tutormfe/plugin.py

```diff
@@ -98,7 +98,7 @@
 # Build, pull and push {mfe}-dev images
 for mfe in ALL_MFES:
     name = f"{mfe}-dev"
-    tag = "{{ DOCKER_REGISTRY }}overhangio/" + mfe + "-dev:{{ MFE_VERSION }}"
+    tag = "{{ DOCKER_REGISTRY }}overhangio/openedx-" + mfe + "-dev:{{ MFE_VERSION }}"
```

Comment:
We need to update `local-docker-compose-dev-services` too.

Reply:
Good catch. I wrote a function so that I stop forgetting about that image.